db/migrate/20220520100819_create_recipe_foods.rb

```diff
@@ -0,0 +1,8 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
```

Comment:
- [OPTIONAL] It would be better to fix database error which I am getting when try to migrate database. Take a look at the screenshot below to see that error :point_down: 
- It says you have 3 pending migrations which I had to delete them all before migrate :point_down:
![Screenshot from 2022-05-21 13-37-55](https://user-images.githubusercontent.com/67087664/169644589-705e8879-6ff9-4c65-9d82-bca06254fca5.png)
![Screenshot from 2022-05-21 13-43-57](https://user-images.githubusercontent.com/67087664/169644625-02004ef0-66cf-47b9-adc5-25b8a240b206.png)

Reply:
### I do not have this problem on my end so I do not know how to go about it.

Here is a snapshot of my database status 🟢 

![image](https://user-images.githubusercontent.com/27709832/169645885-0efd01b6-a020-4b70-86b0-25e16576a4cc.png)